docs/concepts/01_terminologies.md

```diff
@@ -29,7 +29,13 @@ In the [PresentProof](./00_what_is_hl_aries.md#8-presentproof-protocol) flow, th
 ### Issuer
 The entity that issues a [credential](#verifiable-credential) to a holder.
 
-## VDRI
+### Mediator
+
+A mediator is a participant in agent-to-agent message delivery that must be modeled by the sender. It has its own keys and will deliver messages only after decrypting an outer envelope to reveal a forward request.
```

Comment:
imo friendlier wording can be used... along the lines of "it is a router with mailbox features"... and make it a point that it cannot read the encrypted contents of the routed messages.

Reply:
@llorllale that sounds better, i'll update it. i want to to sound as non-tehnical as possible:

```
A mediator is a participant in agent-to-agent message delivery. It can be seen as a router with mailbox features which cannot read the encrypted contents of the routed messages.
```